test/fuzz/fuzzer_pass_flatten_conditional_branches_test.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2020 Google LLC
```

Comment:
My suggestions for moving out the comparator and the helper function would mean that all the content of this file would migrate away elsewhere.